modules/st2-criteria/criteria.component.js

```diff
@@ -51,6 +51,12 @@ const types = {
   'timediff_gt': 'Later Than',
   'exists': 'Exists',
   'nexists': 'Doesn\'t Exist',
+  'search': 'Search',
+};
+
+const searchConditions = {
+  'any': 'Any',
+  'all': 'All',
 };
```

Comment:
There are more search conditions supported, according to https://docs.stackstorm.com/rules.html#advanced-comparison

```suggestion
const searchConditions = {
  'any': 'Any',
  'all': 'All',
  'any2any': 'Any2Any',
  'all2any': 'All2Any',
};
```